libs/cli/pyproject.toml

```diff
@@ -30,6 +30,10 @@ ruff = "^0.1.3"
 
 [tool.poetry.group.typing.dependencies]
 
+[tool.poetry.extras]
+# For langserve
+serve = ["langserve"]
```

Comment:
@eyurtsev here

Reply:
Why is the `[serve]` needed -- is this for use cases of the cli that don't involve serving?

Reply:
Yeah in the future, might just want to install `langchain-cli[smith]` or other submodule deps. Goal is a unified cli.